### PR TITLE
add maven central info to README.md.

### PR DESCRIPTION
**Pull request by bmarwell**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,21 @@ Typically, aspects are used in one of two ways within your Maven reactors:
...
+
+### Maven
+
+This plugin is available [on Maven Central](https://search.maven.org/artifact/dev.aspectj/aspectj-maven-plugin).
+
+```xml
+<plugin>
+  <groupId>dev.aspectj</groupId>
+  <artifactId>aspectj-maven-plugin</artifactId>
+  <version>13.M2</version>
```
Comment (kriegaex) [inline README.md:27]: 1.13.M2

**Comment by kriegaex** [comment]
Before I accept this PR, what do you think about me not adding this information before on purpose, because right below where you added it, there is the plugin documentation link with lots of usage examples. I would like the read-me to be small.

**Comment by bmarwell** [comment]
> not adding this information before on purpose, because right below where you added it, there is the plugin documentation link with lots of usage examples.

This is the ONE info everyone is looking at, imho.

> I would like the read-me to be small.

What is the benefit? Ppl will need to search for information, which is exhausting. I like it "as short as possible, as long as needed". 

I would not include an example, either.

**Comment by kriegaex** [comment]
I thought about it and disagree. There is no exhaustive search here, the read-me is pushing it right into the user's face:

image

Besides, without specifying additional config options, the plugin is normally not usable like in the example anyway.

**Pull request closed without merging**